.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -13,7 +13,8 @@ jobs:
         UPLOAD_PACKAGES: 'True'
   timeoutInMinutes: 360
   variables:
-    CONDA_BLD_PATH: D:\\bld\\
+    CONDA_BLD_PATH: C:\\bld\\
+    SET_PAGEFILE: 'True'
```

Comment:
Is `SET_PAGEFILE: 'True'` still necessary?

Reply:
It doesn't hurt particularly (except taking a few seconds during CI to setup the pagefile), but we can remove this again if you prefer.

Reply:
No need, I was just making sure as I understood it was added to work around a problem that was solved in a different way.